src/ui/common/src/components/workflows/StatusBar.tsx

```diff
@@ -278,18 +278,13 @@ export const WorkflowStatusBar: React.FC<WorkflowStatusBarProps> = ({
   // List of the workflow status items filtered out by category: errors, warnings, logs and checks passed.
   const [listItems, setListItems] = useState<WorkflowStatusItem[]>([]);
 
-  // Ignore Parameters from list of workflow status items
-  // const shouldShowStatusItem = (statusItem: WorkflowStatusItem) => {
-  //   return statusItem.type !== 'paramOp';
-  // };
-
-  const itemsToShow: WorkflowStatusItem[] = workflowStatusItems.filter(
-    (statusItem: WorkflowStatusItem) => {
-      return statusItem.type !== 'paramOp';
-    }
-  );
-
   useEffect(() => {
+    const itemsToShow = workflowStatusItems.filter(
```

Comment:
This file's changes are unrelated to the ENG-1851 task, but take care of a linter warning.